src/map/elemental.cpp

```diff
@@ -280,7 +280,7 @@ int elemental_data_received(struct s_elemental *ele, bool flag) {
 
 int elemental_clean_single_effect(struct elemental_data *ed, uint16 skill_id) {
 	struct block_list *bl;
-	sc_type type = status_skill2sc(skill_id);
+	sc_type type = skill_get_sc(skill_id);
```

Comment:
Not problem of this pull, but should be moved below to prevent useless lookups.

Also from reading the rest of the code this should now check for SC_NONE and cancel if this gets returned.

Reply:
Additionally elemental_clean_single_effect and elemental_clean_effect look like they should be dealt with in the new database too?